gobblin-cluster/src/main/java/org/apache/gobblin/cluster/HelixUtils.java

```diff
@@ -278,9 +278,12 @@ static void waitJobCompletion(HelixManager helixManager, String workFlowName, St
           case STOPPING:
             log.info("Waiting for job {} to complete... State - {}", jobName, jobState);
             Thread.sleep(TimeUnit.SECONDS.toMillis(1L));
+            if (stoppingStateEndTime == 0) {
+              stoppingStateEndTime = currentTimeMillis + stoppingStateTimeoutInSeconds * 1000;
+            }
             // Workaround for a Helix bug where a job may be stuck in the STOPPING state due to an unresponsive task.
-            if (System.currentTimeMillis() > stoppingStateEndTime) {
-              log.info("Deleting workflow {}", workFlowName);
+            if (stoppingStateEndTime != 0 && System.currentTimeMillis() > stoppingStateEndTime) {
```

Comment:
Nit: won't happen b.c. of line 281

Reply:
Updated and changed the stoppingStateEndTime to a Long obj